### DISCUSSION TITLE
How to custom menu item in top nav?

### DISCUSSION THREAD

**Discussion by aimonext** [filamentphp/filament#8355] [General]
## How to add a new custom item after the profile item?
![image](https://github.com/filamentphp/filament/assets/110348549/15542685-e40c-4ed2-9905-9b847bc15c1b)

**Comment by christmex** [comment]
@aimonext  Did you read this?
https://filamentphp.com/docs/3.x/panels/navigation#customizing-the-user-menu


let me know if you find any issues, Cheers!

**Reply by aimonext** [reply]
Thank you so much. I missed it while reading the docs.
